editoast/src/views/infra/mod.rs

```diff
@@ -814,19 +814,13 @@ pub mod tests {
         let empty_infra = empty_infra.await;
         let app = create_test_service().await;
 
-        let switch_type = SwitchType::default();
-        let switch_type_id = switch_type.id.clone();
-        let req = create_object_request(empty_infra.id(), switch_type.into());
-        assert_eq!(call_service(&app, req).await.status(), StatusCode::OK);
-
         let req = TestRequest::get()
             .uri(format!("/infra/{}/switch_types/", empty_infra.id()).as_str())
             .to_request();
         let response = call_service(&app, req).await;
         assert_eq!(response.status(), StatusCode::OK);
         let switch_types: Vec<SwitchType> = read_body_json(response).await;
```

Comment:
You could turn that into a set and compare it with `builtin_node_types_list()` also converted to a set.